pkg/controller/apischeme/apischeme_controller.go

```diff
@@ -145,11 +146,13 @@ func (r *ReconcileAPIScheme) Reconcile(request reconcile.Request) (reconcile.Res
 	reqLogger.Info(fmt.Sprintf("Region: %s, Owner tags: +%v", region, ownerTags))
 	// We expect this secret to exist in the same namespace Account CR's are created
 	// TODO: Get the region of the cluster
-	awsClient, err := awsclient.GetAWSClient(r.client, awsclient.NewAwsClientInput{
-		SecretName: config.AWSSecretName,
-		NameSpace:  config.OperatorNamespace,
-		AwsRegion:  region,
-	})
+	if awsClient == nil {
+		awsClient, err = awsclient.GetAWSClient(r.client, awsclient.NewAwsClientInput{
+			SecretName: config.AWSSecretName,
+			NameSpace:  config.OperatorNamespace,
+			AwsRegion:  region,
+		})
+	}
```

Comment:
I don't know if this is the best way to go about allowing the tests to set the client (line 32). It seems like it could be better.

Reply:
Can't think of a way to make this better but it is annoying that we have to do this